pkgs/misc/vscode-extensions/update_installed_exts.sh

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i sh -p curl jq coreutils
```

Comment:
You are specifying that you use `sh`, but you use keywords like `function`, which are not in `sh`.

Reply:
Which is always `bash` in nixpkgs, but being explicit about `bash` is better though.